core/nativeaot/NativeLibrary/ios-like-native-libraries-with-nativeaot.md

```diff
@@ -0,0 +1,250 @@
+# iOS-like Native Libraries with NativeAOT in .NET 9
```

Comment:
This is samples repo, so there should a working sample that shows you how to do it.

If you do not want to turn this into an actual sample, should this rather be in the docs?

Reply:
Yes, you are right, this fits better in the docs repository.
I created a PR there instead.